### PR TITLE
Iazehaf remove UI directory in no UI mode

### PR DESCRIPTION
**Pull request by iazehaf**

### CODE REVIEW

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst [hunk 1]
```diff
@@ -1,15 +1,24 @@
-#######################################
-Ansys Solutions {{cookiecutter.solution_name}}
-#######################################
+#################################################
+Ansys Solutions {{cookiecutter.solution_name}} 🚀
+#################################################
```
Comment (jorgepiloto) [inline src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst:3]: I am a big fan of icons but I think they are discouraged in technical documentation. Hence, I would suggest to remove it:

```suggestion
##############################################
Ansys Solutions {{cookiecutter.solution_name}}
##############################################
```

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+
+.. note::
+  ⚠️ This content needs to be configured according to the project specifics.
```
Comment (jorgepiloto) [inline src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst:6]:
```suggestion

    This content needs to be configured according to the project specifics. 
```

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst [hunk 3]
```diff
@@ -19,10 +28,15 @@ Prerequisites
 Python version support
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Officially Python 3.7 to 3.8.
+.. note::
+  ✍️ Add here the supported Python versions.
```
Comment (jorgepiloto) [inline src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst:32]:
```suggestion

    Add here the supported Python versions.
```

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst [hunk 4]
```diff
@@ -32,27 +46,63 @@ in the `Solutions Developer's Guide <https://dev-docs.solutions.ansys.com/index.
 
   2. Create a system environment variable named SOLUTIONS_PRIVATE_PYPI_PAT and assign the token.
   
+.. note::
+  🛈 In addition to the Ansys-Solutions private PyPI server, you can add several private sources such as PyAnsys private PyPI server.
```
Comment (jorgepiloto) [inline src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst:50]:
```suggestion

    In addition to the Ansys-Solutions private PyPI server, you can add several private sources such as PyAnsys private PyPI server. 
```

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst [hunk 5]
```diff
@@ -32,27 +46,63 @@ in the `Solutions Developer's Guide <https://dev-docs.solutions.ansys.com/index.
...
   
+.. note::
+  🛈 In addition to the Ansys-Solutions private PyPI server, you can add several private sources such as PyAnsys private PyPI server. 
+
 Flagship products
 ~~~~~~~~~~~~~~~~~
 
-Specify required flagship products.
+.. note::
+  ✍️ Add here the name and the versions of Ansys flagship products needed to run the solution.
```
Comment (jorgepiloto) [inline src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst:56]:
```suggestion

    Add here the name and the versions of Ansys flagship products needed to run the solution. 
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
This looks great. Thanks a lot, @iazehaf. I just left some comments regarding the usage of emojis.

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst (hunk 1 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst]: @jorgepiloto I removed all icons.

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst (hunk 2 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst]: Icon removed.

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst (hunk 3 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst]: Icon removed.

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst (hunk 4 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst]: Icon removed.

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst (hunk 5 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst]: Icon removed.

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**